<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginApp</class>
 <widget class="QMainWindow" name="LoginApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginApp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>[+]Status</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>241</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>SignIn</string>
    </property>
    <widget class="QPushButton" name="Login">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>190</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="username">
        <property name="text">
         <string>UserName</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_username"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>187</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="password">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_password">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
